modules/contact/doc/content/syntax/Contact/index.md

```diff
@@ -87,7 +87,11 @@ large, the solver may struggle due to poor conditioning.
 ## `System` Parameter
 
 The `system` parameter is deprecated and currently defaults to `Constraint`.
-  
+
+## Gap offset parameters
+
+Gap offset can be provided to the current contact formulation enforced using the [MechanicalContactConstraint](/constraints/MechanicalContactConstraint.md). It can be either `secondary_gap_offset` (gap offset from secondary side) or `mapped_primary_gap_offset` (gap offset from primary side but mapped to secondary side). This could be useful to model contact in such as plates or shells. However, the offsetted gap is treated as rigid region without deformation.
```

Comment:
I'd replace the last two sentences with this (I'd drop the part about plates or shells because although that's correct, we don't support that yet, so I wouldn't want to give the impression that we do):

Use of these gap offset parameters treats the surfaces as if they were virtually extended by the specified amount, so that the surfaces are treated as if they are closer than they actually are. There is no deformation within the material in this gap offset region.